base/dump-dotnet2/aws-lambda-tools-defaults.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "Information" : [
+    "This file provides default values for the deployment wizard inside Visual Studio and the AWS Lambda commands added to the .NET Core CLI.",
+    "To learn more about the Lambda commands with the .NET Core CLI execute the following command at the command line in the project root directory.",
+
+    "dotnet lambda help",
+
+    "All the command line options for the Lambda command can be specified in this file."
+  ],
+
+  "profile":"",
+  "region" : "",
+  "configuration" : "Release",
+  "framework" : "netcoreapp2.0",
+  "function-runtime":"dotnetcore2.0",
+  "function-memory-size" : 256,
+  "function-timeout" : 30,
```

Comment:
30 seconds was not enough time for the upload to complete, at least for me. It took around two minutes, but I left it at three minutes just to be safe.